.github/workflows/check_pr.yml

```diff
@@ -118,10 +118,19 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install dependencies
+      - name: Cache node modules
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/docs/node_modules
+          key: node_modules-${{ matrix.os }}-${{ matrix.node }}-${{ hashFiles('**/npm-shrinkwrap.json') }}
```

Comment:
We don't use npm-shrinkwrap.json for docs. Instead, we should change to use package-lock.json